cmd/kube-apiserver/app/server.go

```diff
@@ -127,6 +127,9 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	// Use protobufs for self-communication.
+	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
To clarify - did you mean setting for every loopback config here:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/options/serving.go#L157

?

I don't think it's a good idea, since not every apiserver has to support protobufs.